gamesense/README.md

```diff
@@ -46,6 +46,16 @@ GameSense leverages Parameter-Efficient Fine-Tuning (PEFT) techniques to customi
 - Python 3.8+
 - GPU with at least 24GB VRAM (for full model training)
 - ZenML installed and configured
+- Neptune.ai account for experiment tracking
```

Comment:
should be optional i think?

Reply:
the finetuning step fails if you don't add neptune api key and project ID. could be used internally but not something that i have set myself.

Reply:
That mechs you just have Neptune installed in your virtuanenv right? No need @wjayesh